src/metpy/testing.py

```diff
@@ -23,37 +24,89 @@
 from .deprecation import MetpyDeprecationWarning
 from .units import units
 
-MPL_VERSION = Version(matplotlib.__version__)
 
+def module_version_check(version_spec):
+    """Return comparison between the active module and a requested version number.
 
-def mpl_version_before(ver):
-    """Return whether the active matplotlib is before a certain version.
+    Will also validate specification against package metadata to alert if spec is irrelevant.
 
     Parameters
     ----------
-    ver : str
-        The version string for a certain release
+    version_spec : str
+        Module version specification to validate against installed package. Must take the form
+        of `f'{module_name}{comparison_operator}{version_number}'` where `comparison_operator`
+        must be one of `['==', '=', '!=', '<', '<=', '>', '>=']`.
 
     Returns
     -------
-        bool : whether the current version was released before the passed in one
+        bool : Whether the installed package validates against the provided specification
     """
-    return MPL_VERSION < Version(ver)
+    comparison_operators = {
+        '==': lambda x, y: x == y, '=': lambda x, y: x == y, '!=': lambda x, y: x != y,
+        '<': lambda x, y: x < y, '<=': lambda x, y: x <= y,
+        '>': lambda x, y: x > y, '>=': lambda x, y: x >= y,
+    }
 
+    # Match version_spec for groups of module name,
+    # comparison operator, and requested module version
+    module_name, comparison, version_number = _parse_version_spec(version_spec)
 
-def mpl_version_equal(ver):
-    """Return whether the active matplotlib is equal to a certain version.
+    # Check MetPy metadata for minimum required version of same package
+    _, _, minimum_version_number = _parse_version_spec(_get_metadata_spec(module_name))
+
+    installed_version = Version(version(module_name))
```

Comment:
I was thinking using the `__version__` attribute, but this approach is cleaner. Avoids the need to manually import anything altogether. I like it.

Reply:
That's what I was doing, but came across this when parsing the package metadata, and I liked this too!